product_variant_exclusion/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Product Variant Exclusion
+=========================
+
+This module extends the functionality of product variants to allow you to
+select which combination of product attribute values you want to exclude
+from the product variants.
+
```

Comment:
@grindtildeath I did do a brief review of this when you asked, and thought things looked really good.  Certainly at a technical implementation level I was at the point of reconsidering my own work.

There are a few housekeeping issues, like having all the models in one file as opposed to seperate ones that will need dealing with.  Would be nice to see better tests as well, rather than just a single template creation.

I'll put a few more questions in as I go.  It won't be complete, its late.  Nothing blocking, just trying to understand.